data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -323,13 +323,15 @@ def _prepare_initial_df(self) -> pd.DataFrame:
         ]
 
         columns_to_keep = non_numeric_columns + numeric_columns
-        df = df[columns_to_keep]
+
+        df_copy = df[columns_to_keep].copy()
+
+        df_copy[numeric_columns] = df_copy[numeric_columns].apply(pd.to_numeric)
```

Comment:
@esfoobar-usds @lucasmbrown-usds Have we ever run into issues where we needed to fill NA with 0? I could have also applied the following:

`apply(pd.to_numeric, errors='coerce').fillna(0)`